flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/data/TestRowProjection.java

```diff
@@ -37,19 +41,17 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.apache.iceberg.types.Comparators;
 import org.apache.iceberg.types.Types;
-import org.junit.Assert;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 public class TestRowProjection {
```

Comment:
In the Flink codebase we need to make sure that the same code changes are being applied across all 3 versions. So if you're touching this file here, you would also need to make sure to apply the same changes for the other Flink versions.

This can be either done within the same PR or in an immediate follow-up that backports the same changes to earlier versions